app/setup.py

```diff
@@ -103,7 +102,7 @@ def add_service(_, __, event_dict):
         renderer_processor,
     ]
     structlog.configure(
-        context_class=wrap_dict(dict),
+        context_class=dict,
```

Comment:
Probably one for a different story, this is the old way of using structlog and isn't thread save. I have no idea what removing the wrap will do in regards to how the logs look, however I also think if we touching this we should do it properly. Should be like this now https://github.com/ONSdigital/ras-frontstage/pull/918/files